src/module.tsx

```diff
@@ -40,7 +40,7 @@ export const plugin = new PanelPlugin<SimpleOptions>(ImageItPanel).setPanelOptio
       description: 'List of sensors',
       category: ['Sensors'],
       defaultValue: [],
-      editor: props => {
+      editor: function SensorListEditor(props) {
```

Comment:
Why changing the arrow function?

Reply:
That was suggested by the grafana toolkit